docs/guides/cli.md

```diff
@@ -35,6 +35,12 @@ Connection: keep-alive
 
 Responses will be mocked using realistic data that conforms to the type in the description.
 
+### Auto-reloading
+
+Prism watches for changes made to a document it was loaded with.
+When they happen, Prism restarts its HTTP server to reflect changes to operations.
```

Comment:
Can we also explicit what happens if the target (modified) document is incorrect?

Reply:
FYI — In markdown you need to use two new lines to make an actual new line :trollface: 

This is how the docs look like at the moment: 

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/1416224/66577480-c1698900-eb79-11e9-98c7-792b76f3eacf.png">

Reply:
It was intended ;> Just added line breaks so that if you read it in a text/code editor, you have them :)

Reply:
Can we break this in at least two small paragraphs?

Reply:
Broke it into two paragraphs. The first one is about the fact Prism now has hot-reloading and the second one is about when Prism is not gonna auto-reload.